### PR TITLE
Name meme

### PR DESCRIPTION
**Pull request by TheDavestDave**
I couldn't resist and I'm not even sorry

### CODE REVIEW

**Comment by Detective-Google** [comment]
Once again "first aider" is not grammatically correct. At all.

**Comment by TheDavestDave** [comment]
that's not even the joke here

File: code/game/jobs/job/exploration_vr.dm [hunk 1]
```diff
@@ -102,14 +108,20 @@
...
 /datum/alt_title/surveyor
 	title = "Surveyor"
 
 /datum/alt_title/offsite_scout
 	title = "Offsite Scout"
 
+/datum/alt_title/offsite_support
+	title = "Offsite Support"
+
+/datum/alt_title/steve_irwin
```
Comment (TheDavestDave) [inline code/game/jobs/job/exploration_vr.dm:122]: joke is here

File: code/game/jobs/job/exploration_vr.dm [hunk 2]
```diff
@@ -102,14 +108,20 @@
...
+	title = "Wild Life Defender"
```
Comment (TheDavestDave) [inline code/game/jobs/job/exploration_vr.dm:123]: the joke is here

**Comment by Very-Soft** [comment]
No.

**Pull request closed without merging**